apps/mail/app/globals.css

```
@@ -12,16 +12,19 @@
 @plugin "tailwindcss-animate";
 @plugin "tailwind-scrollbar" {
   nocompatible: true;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)